Update README.md (#249)
AdobeDocs/commerce-webapi: README.md

## Local development

This is a [Gatsby](https://www.gatsbyjs.com/) project that uses the [Adobe I/O Theme](https://github.com/adobe/aio-theme).

> Ensure that your local environment matches the prerequisites for the Adobe AIO-Theme.

Reviewer: ```suggestion
Ensure that your local environment matches the prerequisites described in the Adobe I/O Theme README.
```
